config/routes.rb

```diff
@@ -3,7 +3,7 @@
   mount Blacklight::Oembed::Engine, at: 'oembed'
   mount Riiif::Engine => '/images', as: 'riiif'
   root to: 'spotlight/exhibits#index'
-  mount Spotlight::Engine, at: 'spotlight'
+  mount Spotlight::Engine, at: '/'
```

Comment:
I have not seen any yet:

http://exhibits.library.cornell.edu/jenn-test/catalog/3-1

This is what I was looking at from Stanford:

https://github.com/sul-dlss/exhibits/blob/master/config/routes.rb